### ISSUE TITLE
migrate from angular 11 to angular 12

### ISSUE THREAD

**Issue by selouali** [nrwl/nx#5997]
Hello
I try to migrate from angular 11 to angular 12 using the nx migrate command, but I've an error I can't understand what happens and how to fix it.
when I run `nx migrate latest` my output is :

```
/Users/USERNAME/Projects/myNxAngular11Project/node_modules/yargs/build/lib/yargs.js:1132
                throw err;
                ^

Error: Command failed: npm install --legacy-peer-deps
    at checkExecSyncError (child_process.js:635:11)
    at Object.execSync (child_process.js:671:15)
    at taoPath (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/@nrwl/workspace/src/command-line/nx-commands.js:312:21)
    at Object.handler (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/@nrwl/workspace/src/command-line/nx-commands.js:86:19)
    at Object.runCommand (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/yargs/build/lib/command.js:196:48)
    at Object.parseArgs [as _parseArgs] (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/yargs/build/lib/yargs.js:1043:55)
    at Object.get [as argv] (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/yargs/build/lib/yargs.js:986:25)
    at Object.initLocal (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/@nrwl/cli/lib/init-local.js:24:13)
    at Object.<anonymous> (/Users/USERNAME/Projects/myNxAngular11Project/node_modules/@nrwl/cli/bin/nx.js:43:18)
    at Module._compile (/Users/USERNAME/.nvm/versions/node/v12.21.0/lib/node_modules/@nrwl/cli/node_modules/v8-compile-cache/v8-compile-cache.js:192:30) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 75743,
  stdout: null,
  stderr: null
}
```

I don't know why and how to fix this issue. I tryied npm 6.14.13 and 7.16.0, with node 12.21.0 and 14.17.0. (with rm -fr ./node_modules at each version change of node/npm)

my nx report

```
>  NX  Report complete - copy this into the issue template

  Node : 12.21.0
  OS   : darwin x64
  npm  : 7.16.0
  
  nx : Not Found
  @nrwl/angular : 12.0.8
  @nrwl/cli : 12.0.8
  @nrwl/cypress : 12.0.8
  @nrwl/devkit : 12.0.8
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 12.0.8
  @nrwl/linter : 12.0.8
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 12.0.8
  @nrwl/web : Not Found
  @nrwl/workspace : 12.0.8
  @nrwl/storybook : Not Found
  @nrwl/gatsby : Not Found
  typescript : 4.1.4
```

What should I do to fix that and be able to run migration ? which version should I use ?

**Comment by jredl-va** [comment]
Note, I experienced something similar when moving to angular 12. Note that I performed the update with the angular tooling itself. 

```
FORCE_NG_UPDATE=true ng update @angular/cli @angular/core --force
```

**Comment by selouali** [comment]
Seems to be fixed with the new 12.4.0 cli version.
